.storybook/preview.js

```diff
@@ -18,13 +19,28 @@ async function run() {
     addDecorator(withA11y);
     addDecorator(withKnobs);
     addDecorator(withWebComponentsKnobs);
-    addDecorator(
-        (story) => html`
-            <sp-theme id="root-theme" color="light" scale="medium">
+    const colorOptions = {
+        Lightest: 'lightest',
+        Light: 'light',
+        Dark: 'dark',
+    };
+    let defaultColor = colorOptions.Light;
+    const scaleOptions = {
+        Medium: 'medium',
+        Large: 'large',
+    };
+    let defaultScale = scaleOptions.Medium;
+    addDecorator((story) => {
+        const color = select('Color', colorOptions, defaultColor, 'Theme');
```

Comment:
This immediately surfaced an issue in the Action Button pattern, so I've created #435 to track reviewing the patterns across the various theme settings. Whatever is broken was broken before, so I'd like to be able to merge this without that work, but can be convinced.

Once we get the styles up to snuff, we can also add separate passes on the Visual Regressions to ensure we get keep those styles stable across future changes. Having more than one list would also mark a good time to alter the test collection pattern there as the array of strings is pretty fragile and our recent move to CSF should give us the chance to leverage that approach there. I've created #436 to track that work.